templates/user_form.tt.html

```diff
@@ -7,32 +42,101 @@
     <!-- Start form -->
 
     <form method="post" action="/cgi/user.pl" enctype="multipart/form-data">
-        <table>
-            [% display_user_form %]
-            [% display_user_form_optional %]
-            [% display_user_form_admin_only %]
-
-            [% IF admin %]
-                <tr><td colspan="2">
-					<label>
-						<input type="checkbox" name="delete" onchange="if (this.checked) { if (!confirm('This action cannot be undone, are you sure you want to delete the user?')) { this.checked = false }}" />
-						[% lang("delete_user") %]
-					</label>
-                </td></tr>
+		[% FOREACH section IN sections %]
+
+            [% IF section.name %]
+                <fieldset id="[% section.id %]_section">
+                    <legend [% IF section.id == "administrator" %] style = "background-color:#ffcccc"[% END %]>[% section.name %]</legend>
+            [% END %]
+
+            [% IF section.description %]
+                <p >[% section.description %]</p>
+            [% END %]
+
+            [% IF section.note %]
+                <p >[% section.note %]</p>
+            [% END %]
+
+            [% FOREACH field IN section.fields %]
+                [% IF field.field == 'userid' %]
+                    [% IF type == 'edit' %]
+                        <label for="[% field.field %]"> [% lang(field.label) %]</label>
+                        <p> [% field.value %] </p>
```

Comment:
```suggestion
                        <p id="userid">[% field.value %]</p>
```

Reply:
in fact there's already an hidden field with the userid, so maybe I can add id="userid" to it, to make it consistent with the other fields.

        [% IF userid %]
            <input type="hidden" name="userid" value="[% userid %]" />
            <input type="submit" name=".submit" class="button" value="[% lang("save") %]" />
        [% ELSE %]